docs/src/less/base/index.less

```diff
@@ -233,7 +233,7 @@ span.demo__inner {
 /* CAROUSEL SECTION */
 
 .carousel__list--default-demo li {
-    background-color: @color-information-background;
+    background-color: @color-status-information;
```

Comment:
This color looks off on the carousel and doesn't have a good color contrast on it. 
<img width="1169" alt="Screen Shot 2020-09-11 at 11 43 09 AM" src="https://user-images.githubusercontent.com/1755269/92961375-fc2c0f00-f423-11ea-94be-a3cd66fc729a.png">

Reply:
I think it's currently like this in production, right? If so, we should fix, but in a different PR.

Reply:
Oh scrap that - I see i used the wrong variable. Will fix.

Reply:
Fixed. Good spot!

![Screen Shot 2020-09-11 at 11 54 13 AM](https://user-images.githubusercontent.com/38065/92962252-8d4fb580-f425-11ea-844e-0dc9edd19c68.png)